user/migrate/legacy-services-to-github-apps-migration-guide.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Legacy GitHub Services to GitHub Apps Migration Guide
+layout: en
+redirect_from: user/travis-migrate-to-apps-gem-guide/
+---
+
+As of May 2, 2018 we are moving toward having repositories integrated via a GitHub App instead of GitHub Services which [will no longer be supported by GitHub as of January 31st, 2018](https://developer.github.com/v3/guides/replacing-github-services/#deprecation-timeline).
```

Comment:
```suggestion
As of May 2, 2018 we are moving toward having repositories integrated via a GitHub App instead of GitHub Services which [will no longer be supported by GitHub as of January 31st, 2019](https://developer.github.com/v3/guides/replacing-github-services/#deprecation-timeline).
```

Happy new year! hahahaha.